parliament_challenge.py

```diff
@@ -0,0 +1,111 @@
+import requests
+import waitress
+import flask
+from flask import Flask, request, json
+
+
+def filter_speeches_dict(speeches):
+    """
+    Filter speeches data for relevant items.
+    """
+
+    speeches_list = []
+    for s in speeches:
+        speechesDict = {
+            "anforande_id": s.get("anforande_id"),
+            "dok_datum": s.get("dok_datum"),
+            "parti": s.get("parti"), 
+            "avsnittsrubrik": s.get("avsnittsrubrik"),
+            "links": [{ 
+                "rel": "speech",
+                "href": s.get("protokoll_url_www")
+                    }],
+            "intressent_id": s.get("intressent_id")
+            }
+        speeches_list.append(speechesDict)
+    return speeches_list
+
+
+def filter_member_dict(member):
+    """
+    Filter member data for relevant items.
```

Comment:
This is not correct. You're filtering speech data for the relevant "keys", not items.

And also say that the relevant keys are `tilltalsnamn`, `valkrets` .. and so on ..

Reply:
updated:
    Filtering member data for relevant keys.
    Relevant keys are tilltalsnamn, valkrets,
    bild_url_192 and intressent_id.